docs/GETTING_STARTED.md

```diff
@@ -193,6 +193,21 @@ and on [storybook](https://lyne-storybook.app.sbb.ch).
 
 ## Styles
 
+### Lean variant
+
+Lean uses a more compact design. To activate it, add the CSS class `sbb-lean` on the document's body tag.
```

Comment:
I think, it would be a good idea to document more exactly what happens. like choses size s as default size whenever there is a size property